tfx/components/transform/executor_test.py

```diff
@@ -166,6 +169,28 @@ def testDoWithDuplicatePreprocessingFn(self):
       self._transform_executor.Do(self._input_dict, self._output_dict,
                                   self._exec_properties)
 
+  def testDoWithCustomSplits(self):
+    self._exec_properties['splits_config'] = json_format.MessageToJson(
+        transform_pb2.SplitsConfig(analyze_splits=['train'], 
+                                   transform_splits=['eval']),
+        preserving_proto_field_name=True)
+    self._exec_properties['module_file'] = self._module_file
+    self._transformed_examples.split_names = artifact_utils.encode_split_names(
+        ['eval'])
+    self._output_dict[executor.TRANSFORMED_EXAMPLES_KEY] = [self._transformed_examples]
```

Comment:
format?

Reply:
you mean pylint format issues? I added another commit to fix all the pylint issues. this is an older version.